lib/broccoli_sass_compiler.js

```diff
@@ -485,7 +485,7 @@ BroccoliSassCompiler.prototype.handleCacheHit = function(details, inputAndOutput
 BroccoliSassCompiler.prototype.scopedFileName = function(file) {
   file = this.relativize(file);
   if (this.treeName) {
-    return this.treeName + "/" + file;
+    return this.treeName + path.sep + file;
```

Comment:
Sorry for abandoning this PR, don't have an env suitable for testing anymore 
You could probably apply a change, proposed above and test, I don't think it is the only place where paths are concatenated as strings, but it will be enough to make this PR accepted, hopefully.